assets/js/main.js

```diff
@@ -23,11 +23,11 @@ const createParticipantes = (listOfParticipants) => {
   listOfParticipants.forEach(async (participant) => {
     const cardHTML = `
       <h2 class="card-title">${await getRandomEmoji()} ${participant.name}</h2>
-      <p class="card-sub margin-yt-sm">📅 Started: ${participant.course_start}</p>
-      <p class="card-sub margin-yb-sm custom-underline">🎓 Stage:  ${participant.course_stage}</p>
-      <p class="card-detail">Loves: ${participant.favorite_language} 😍 </p>
-      <p class="card-detail custom-underline">Learning: ${participant.currently_learning} 📚 </p>
-      <a class="participant-link" href="community/${participant.name}" data-content="View Work ➡">View Work ➡</a>
+      <h3 class="card-sub margin-yt-sm">📅 Started: ${participant.course_start}</h3>
+      <h3 class="card-sub margin-yb-sm custom-underline">🎓 Stage:  ${participant.course_stage}</h3>
+      <h3 class="card-detail">Loves: ${participant.favorite_language} 😍 </h3>
+      <h3 class="card-detail custom-underline">Learning: ${participant.currently_learning} 📚 </h3>
+      <a class="participant-link" href="community/${participant.name}" data-content= "">${participant.name}'s Work ➡</a>
```

Comment:
Hi @puritybirir the `data-content` attribute needs to have the same value as the actual rendered HTML. It is being used to promote a `content` CSS property animation on hover.
```html
<a class="participant-link" href="community/${participant.name}" data-content="${participant.name}'s Work ➡">${participant.name}'s Work ➡</a>
```

Consideration for the above to be implemented, and then we can Approve and Merge? 😄 Very happy to discuss on this and will definitely take into account your professional stake in it too. The animation is part of the embedded theme and while not necessarily fulfilling the Accessibility role, it is fulfilling the creativeness of the web in a unique way. 🤜

Reply:
Hey @auxfuse ,  I've done that. I'd initially changed it because I figured having more specific text would be a better experience. I understand completely and agree. I've reverted back, should be good now.

Reply:
Also, the original data content still passes all accessibility checks and we are still at 0 alerts.

Reply:
@puritybirir Thank you very much for helping us with this PR. It has been a pleasure! 🤜 🙌